### PR TITLE
Update enrichers catalog

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: src/app/custom/translations.tsv [hunk 1]
```diff
@@ -720,14 +720,14 @@
...
-"ws_post-v1-homogenize-publisher-json_description"	"Brings different forms of writing from the publisher to a standardized form."	"Ramène différentes formes d’écriture d'un éditeur à une forme standardisée."
+"ws_post-v1-rnsr-json_title"	"Assigning RNSR identifier(s) to an address (Alignments)"	"Attribution d’identifiant(s) RNSR à une adresse (Alignements)"
+"ws_post-v1-rnsr-json_description"	"Assigns, using rules, one or more RNSR identifiers based on an author affiliation address and a year of publication."	"Attribue, à l’aide de règles, un ou plusieurs identifiants RNSR à partir d’une adresse d’affiliation d’auteur et d’une année de publication."
+"ws_post-v1-halauthorid-idref-json_title"	"Associate an Author IDREF with a HAL Author ID"	"Associer un IDREF Auteur à un identifiant HAL Author ID"
+"ws_post-v1-halauthorid-idref-json_description"	"Allows to make correspond the identifier IDREF authors to that of the publishers in HAL, having an identifier IDAHL authors."	"Permet de faire correspondre l’identifiant IDREF auteurs à celui des publiants dans HAL, disposant d’un identifiant IDAHL authors."
```
Comment (parmentf) [inline src/app/custom/translations.tsv]: **typo:** ‏
```suggestion
"ws_post-v1-halauthorid-idref-json_description"	"Allows to make correspond the identifier IDREF authors to that of the publishers in HAL, having an identifier IDHAL authors."	"Permet de faire correspondre l’identifiant IDREF auteurs à celui des publiants dans HAL, disposant d’un identifiant IDHAL authors."
```

Comment (parmentf) [reply src/app/custom/translations.tsv]: **polish:** ‏English translation... feels not idiomatic

image

File: src/app/custom/translations.tsv [hunk 2]
```diff
@@ -720,14 +720,14 @@
...
+"ws_post-v1-homogenize-document-type-json_description"	"Allows the ""document type"" values in a bibliographic record from the CONDITOR database (data from HAL, Crossref, Pubmed) to be written in the same way as the ""document type"" values from the Web of Science (for 2019). "	"Permet d’homogénéiser les formes d’écriture des valeurs “types de documents” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed), à partir des valeurs “types de document” issues du Web of Science (sur l’année 2019)."
+"ws_post-v1-homogenize-publisher-json_title"	"Homogenisation of publishers"	"Homogénéisation des éditeurs"
+"ws_post-v1-homogenize-publisher-json_description"	"Used to standardise the way ""publisher"" values are written in a bibliographic record, taken from the CONDITOR database (data from HAL, Crossref, Pubmed) using ""publisher"" values taken from the Web of Science (for 2019)."	"Permet d’homogénéiser les formes d’écriture des valeurs “éditeurs” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed) à partir de valeurs “éditeurs” issues du Web of Science (sur l’année 2019)."
```
Comment (parmentf) [inline src/app/custom/translations.tsv]: **polish:** ‏Other English texts seem to be in american English (see line 731).

```suggestion
"ws_post-v1-homogenize-document-type-json_title"	"Homogenization of document types"	"Homogénéisation des types de documents"
"ws_post-v1-homogenize-document-type-json_description"	"Allows the ""document type"" values in a bibliographic record from the CONDITOR database (data from HAL, Crossref, Pubmed) to be written in the same way as the ""document type"" values from the Web of Science (for 2019). "	"Permet d’homogénéiser les formes d’écriture des valeurs “types de documents” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed), à partir des valeurs “types de document” issues du Web of Science (sur l’année 2019)."
"ws_post-v1-homogenize-publisher-json_title"	"Homogenization of publishers"	"Homogénéisation des éditeurs"
"ws_post-v1-homogenize-publisher-json_description"	"Used to standardize the way ""publisher"" values are written in a bibliographic record, taken from the CONDITOR database (data from HAL, Crossref, Pubmed) using ""publisher"" values taken from the Web of Science (for 2019)."	"Permet d’homogénéiser les formes d’écriture des valeurs “éditeurs” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed) à partir de valeurs “éditeurs” issues du Web of Science (sur l’année 2019)."
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: src/app/custom/translations.tsv (hunk 2 above)
Comment (AlasDiablo) [reply src/app/custom/translations.tsv]: I use UK English and i will do the same to all ws, but lodex already have a mix of US and UK English
- https://www.wordreference.com/enfr/Homogenisation
- https://www.wordreference.com/enfr/standardise

**Review by touv — DISMISSED** [review]

**Comment by touv** [comment]
@AlasDiablo  Is it still in draft ?

**Comment by AlasDiablo** [comment]
> @AlasDiablo Is it still in draft ?

Yes I have completed 3/4 of the file, still 1/4 to do.

File: src/app/custom/enrichers/enrichers-catalog.json [hunk 3]
```diff
@@ -66,31 +66,31 @@
...
+        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/terms-extraction#v1%2fteeft%2fen",
         "swagger": "https://openapi.services.istex.fr/?urls.primaryName=terms-extraction%20-%20Extraction%20de%20termes#/terms-extraction/post-v1-teeft-en",
         "objectifTDM": "https://services.istex.fr/2021/12/20/extraction-de-termes-teeft/",
         "type": "indexation"
     },
     {
         "id": "post-v1-teeft-fr",
         "url": "https://terms-extraction.services.istex.fr/v1/teeft/fr",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/terms-extraction#v1%2fteeft%2fwith-numbers%2fen",
+        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/terms-extraction#v1%2fteeft%2ffr",
```
Comment (leogail) [inline src/app/custom/enrichers/enrichers-catalog.json:77]: **praise (non-blocking):** well seen

File: src/app/custom/translations.tsv [hunk 4]
```diff
@@ -720,44 +720,44 @@
...
+"ws_post-v1-rnsr-instituts-cnrs-json_description"	"Assigns a CNRS institute, based on an RNSR identifier, in which a French research structure is located."	"Attribue, à partir d’un identifiant RNSR, un institut CNRS, dans lequel se trouve une structure de recherche française."
+"ws_post-v2-crossref-prefixes-expand_title"	"Verbalisation of DOI prefixes"	"Verbalisation des préfixes de DOI "
+"ws_post-v2-crossref-prefixes-expand_description"	"Queries the Crossref database and reads the prefix of a DOI from the identifier prefix or the full DOI identifier."	"Interroge la base de données Crossref et verbalise le préfixe d’un DOI à partir du préfixe de l’identifiant ou de l’identifiant DOI complet."
+"ws_post-v1-teeft-en_title"	"Extraction of terms from an English text using Teeft"	"Extraction de termes d’un texte en Anglais via Teeft"
+"ws_post-v1-teeft-en_description"	"Extracts, by default, the 5 most specific terms from an English text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en Anglais."
+"ws_post-v1-teeft-fr_title"	"Extraction of terms from an French text using Teeft"	"Extraction de termes d’un texte en Français via Teeft"
```
Comment (leogail) [inline src/app/custom/translations.tsv]: **praise:** 
```suggestion
"ws_post-v1-teeft-fr_title"	"Extraction of terms from a French text using Teeft"	"Extraction de termes d’un texte en Français via Teeft"
```

File: src/app/custom/translations.tsv [hunk 5]
```diff
@@ -720,44 +720,44 @@
...
+"ws_post-v1-teeft-fr_description"	"Extracts, by default, the 5 most specific terms from an French text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en Français."
+"ws_post-v1-teeft-with-numbers-en_title"	"Extraction of terms from an English text using Teeft (including numbers)"	"Extraction de termes d’un texte en Anglais via Teeft (nombres compris)"
+"ws_post-v1-teeft-with-numbers-en_description"	"Extracts the most relevant terms from a text in English or French, retaining the numbers: it provides terms containing numbers (important when dealing with chemical formulae, physical quantities, etc.)."	"Extrait les termes les plus pertinents d’un texte en Anglais ou en français, en conservant les chiffres : il fournit des termes contenant des chiffres (c’est important quand on a des formules chimiques, des grandeurs physiques, …)."
+"ws_post-v1-addresses-parse_title"	"Address breakdown"	"Décomposition d'une adresse"
```
Comment (leogail) [inline src/app/custom/translations.tsv:745]: **thought (non-blocking):** don't know how to translate this properly but François approved it, so I do

**Review by leogail — APPROVED** [review]

**Review by parmentf — APPROVED** [review]

File: src/app/custom/translations.tsv [hunk 6]
```diff
@@ -720,44 +720,44 @@
...
+"ws_post-v1-teeft-en_description"	"Extracts, by default, the 5 most specific terms from an English text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en Anglais."
+"ws_post-v1-teeft-fr_title"	"Extraction of terms from a French text using Teeft"	"Extraction de termes d’un texte en Français via Teeft"
+"ws_post-v1-teeft-fr_description"	"Extracts, by default, the 5 most specific terms from an French text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en Français."
+"ws_post-v1-teeft-with-numbers-en_title"	"Extraction of terms from an English text using Teeft (including numbers)"	"Extraction de termes d’un texte en Anglais via Teeft (nombres compris)"
+"ws_post-v1-teeft-with-numbers-en_description"	"Extracts the most relevant terms from a text in English or French, retaining the numbers: it provides terms containing numbers (important when dealing with chemical formulae, physical quantities, etc.)."	"Extrait les termes les plus pertinents d’un texte en Anglais ou en français, en conservant les chiffres : il fournit des termes contenant des chiffres (c’est important quand on a des formules chimiques, des grandeurs physiques, …)."
```
Comment (parmentf) [inline src/app/custom/translations.tsv]: **typo:** In French, languages are not capitalized
‏
```suggestion
"ws_post-v1-teeft-en_title"	"Extraction of terms from an English text using Teeft"	"Extraction de termes d’un texte en anglais via Teeft"
"ws_post-v1-teeft-en_description"	"Extracts, by default, the 5 most specific terms from an English text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en anglais."
"ws_post-v1-teeft-fr_title"	"Extraction of terms from a French text using Teeft"	"Extraction de termes d’un texte en français via Teeft"
"ws_post-v1-teeft-fr_description"	"Extracts, by default, the 5 most specific terms from an French text."	"Extrait, par défaut, les 5 termes les plus spécifiques d’un texte en français."
"ws_post-v1-teeft-with-numbers-en_title"	"Extraction of terms from an English text using Teeft (including numbers)"	"Extraction de termes d’un texte en anglais via Teeft (nombres compris)"
"ws_post-v1-teeft-with-numbers-en_description"	"Extracts the most relevant terms from a text in English or French, retaining the numbers: it provides terms containing numbers (important when dealing with chemical formulae, physical quantities, etc.)."	"Extrait les termes les plus pertinents d’un texte en anglais ou en français, en conservant les chiffres : il fournit des termes contenant des chiffres (c’est important quand on a des formules chimiques, des grandeurs physiques, …)."
```

File: src/app/custom/translations.tsv [hunk 7]
```diff
@@ -799,18 +799,18 @@
...
-"ws_post-v1-gotagger_title"	"Detection of geographical entities"	"Détection d’entités géographiques"
-"ws_post-v1-gotagger_description"	"Detects geographical features in an english text."	"Détecte les entités géographiques d'un texte en anglais."
+"ws_post-v1-perseeBaipTagger_title"	"Detection of named entities in public education administrative bulletins - Persée"	"Détection d’entités nommées dans les bulletins administratifs de l’instruction publique – Persée"
+"ws_post-v1-perseeBaipTagger_description"	"Detects specific named entities in the ""bulletins administratifs de l’instruction publique"" (BAIP), Persée."	"Détecte des entités nommées spécifiques dans les bulletins administratifs de l’instruction publique (BAIP), Persée."
+"ws_post-v1-gotagger_title"	"Extraction of geographical entities"	"Extraction d’entités géographiques"
+"ws_post-v1-gotagger_description"	"Detects geographical entities in English text and returns a list of locations found in the text."	"Détecte des entités géographiques dans un texte en Anglais et renvoie la liste des localisations trouvées dans ce texte."
```
Comment (parmentf) [inline src/app/custom/translations.tsv]:
```suggestion
"ws_post-v1-gotagger_description"	"Detects geographical entities in English text and returns a list of locations found in the text."	"Détecte des entités géographiques dans un texte en anglais et renvoie la liste des localisations trouvées dans ce texte."
```

**Review by parmentf — CHANGES_REQUESTED** [review]

**Review by parmentf — APPROVED** [review]

**Review by touv — APPROVED** [review]

**Pull request merged**